query_safety_filter.py

```
@@ -0,0 +1,274 @@
+        'crack': 0.3,
+        
+        # Moderate risk (ε += 0.2)
+        'manipulate': 0.2,
+        'deceive': 0.2,
+        'trick': 0.2,
+        'evade': 0.2,
+        
+        # Low risk (ε += 0.1)
+        'controversial': 0.1,
+        'sensitive': 0.1,
+    }
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)